st2api/tests/unit/test_validation_utils.py

```diff
@@ -48,14 +48,12 @@ def test_validate_auth_cookie_is_correctly_configured_error(self):
         invalid_values = ["strictx", "laxx", "nonex", "invalid"]
 
         for value in invalid_values:
-            cfg.CONF.set_override(
-                group="api", name="auth_cookie_same_site", override=value
-            )
-
-            expected_msg = "Valid values are: strict, lax, none, unset"
-            self.assertRaisesRegex(
-                ValueError, expected_msg, validate_auth_cookie_is_correctly_configured
```

Comment:
The newer version of oslo.config errors out itself, so some of our custom validation in the `validate_auth_cookie_is_correctly_configured` function is no longer needed.